docusaurus/src/theme/Admonition/CustomAdmonition.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import clsx from 'clsx';
+import { ThemeClassNames } from '@docusaurus/theme-common';
+
+const defaultClassName = ThemeClassNames.common.admonition;
+const customDefaultProps = {
+  note: {
+    icon: '✏️',
+    title: 'Note',
+  },
+  tip: {
+    icon: '💡',
+    title: 'Tip',
+  },
+  info: {
+    icon: '👀',
+    title: 'Info',
+  },
+  caution: {
+    icon: '✋',
+    title: 'Caution',
+  },
+  warning: {
+    icon: '❗️',
+    title: 'Warning',
```

Comment:
It makes sense to me to have warning and danger use the same params (icon, title), but I believe warning is rendered with a yellow-ish background, just like caution, while danger has a red one (see `admonition.scss`).

Could you please update the selectors to match: caution and warning → yellow, danger → red? 🙂

Reply:
😅
This one I didn't understand properly, PiWi.

---

`:::caution` is yellow-ish already, look:

![Screenshot 2023-03-09 at 2 05 38 PM](https://user-images.githubusercontent.com/4141420/224104260-f974f53c-1aee-46f5-9f52-e61e04932b7b.png)

---

`:::warning` is yellow-ish already too:

![Screenshot 2023-03-09 at 2 05 00 PM](https://user-images.githubusercontent.com/4141420/224104253-b7a62a44-2020-4071-918c-007d7da4145d.png)

_But I believe we could replace the Warning icon to "⚠️"_

---

I did not found any `:::danger` in the documentation, so I temporarily improvised this one:

![Screenshot 2023-03-09 at 2 17 14 PM](https://user-images.githubusercontent.com/4141420/224105033-e8b0ff5c-7319-4129-8603-33665c333834.png)

---

Reply:
Yes, we don't use :::danger but I think it's cool we keep it as a fallback (so it should behave exactly like :::warning)

We can indeed replace the icon.
What I'd also like is that Warning and Danger are both red, not yellow :)